en/docs/learn/consent-amendment-history.md

```diff
@@ -0,0 +1,13 @@
+Consent Amendment history is a feature that is supported in the CDS Toolkit, and any change that is performed on an 
```

Comment:
Suggestions:
Any change that is performed on an active consent is considered as an amendment. WSO2 Open Banking CDS Toolkit provides the Consent Amendment History feature to utilize and retrieve details related to the consent amendment history.

Let's use active voice as much as possible.